### DISCUSSION TITLE
AI / LLM Coding Assistance

### DISCUSSION THREAD

**Discussion by daley-mottley** [dmotts/yt-to-cloud-uploader#24] [Polls]
Leave a comment on what platforms or tools you used to write or assist your coding / designing / documentation?

**Comment by daley-mottley** [comment]
Let me get the ball rolling ⚽ by saying ChatGPT overall! 😄

The features that vanilla ChatGPT offers are all you need in my opinion.

Happy coding everyone! 🧑‍💻🙌

**Comment by HomerusJa** [comment]
In my opinion, claude vastly outperforms chatgpt

**Reply by daley-mottley** [reply]
Oh wow, I forgot Claude existed 🤣 I ...Lol...I'm going try it I when can... I used it maybe twice since it came out.
